tests/system/examples/dask/test_dask.py

```diff
@@ -33,6 +31,9 @@
 from mlrun_pipelines.common.models import RunStatuses
 from tests.system.base import TestMLRunSystem
 
+kfp = pytest.importorskip("kfp")
+kfp.compiler = pytest.importorskip("kfp.complier")
```

Comment:
I prefer to skip tests from the `make` command rather than skip them all together - what you did here will cause tests to skip for enterprise tests in cause we missed installation of the kfp